### PR TITLE
Code and json for textbook thumbnail navigated event

### PR DESCRIPTION
**Pull request by arslanhashmi**
**Trello Link:** [here](https://trello.com/c/NodskS2V/131-textbook-interaction-event-textbookpdfthumbnailnavigated)

**Description:** The browser emits textbook.pdf.thumbnail.navigated events when a user clicks on a thumbnail image to navigate to a page.
**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/textbook.pdf.thumbnail.navigated.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://localhost:18000/u/edx",
+    "name": "edx",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/textbook.pdf.thumbnail.navigated.json]: Please use a more descriptive username.

File: openedx/features/caliper_tracking/transformers/textbook_transformers.py [hunk 2]
```diff
@@ -198,3 +198,41 @@ def book(current_event, caliper_event):
...
+
+    caliper_event['referrer']['type'] = 'WebPage'
+
+    caliper_event['extensions']['extra_fields'].update({
+        'course_id': current_event['context'].get('course_id'),
+        'ip': current_event['ip'],
+    })
+
+    current_event_details = json.loads(current_event['event'])
+    current_event_details.pop('name')
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/textbook_transformers.py]: Why are you popping name here?

Comment (arslanhashmi) [reply openedx/features/caliper_tracking/transformers/textbook_transformers.py]: Redundancy.

Comment (asamolion) [reply openedx/features/caliper_tracking/transformers/textbook_transformers.py]: Oh, cool.

**Pull request merged**